python/Dockerfile-3.9

```
@@ -10,105 +10,63 @@ ENV LANG C.UTF-8
 # Set shell
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-# Setup requirements
+# extra dependencies (over what buildpack-deps already includes)
 RUN \
-  apt-get update && apt-get install -y --no-install-recommends \
-    ffmpeg \
-    libavcodec-dev \
-    libavdevice-dev \
-    libavfilter-dev \
-    libavformat-dev \
-    libavutil-dev \
-    libswresample-dev \
-    libswscale-dev \
-    libudev-dev \
-  && rm -rf /var/lib/apt/lists/*
+    apt-get update && apt-get install -y --no-install-recommends \
+        libbluetooth-dev \
+        tk-dev \
+        uuid-dev \
+        ffmpeg \
+        libavcodec-dev \
+        libavdevice-dev \
+        libavfilter-dev \
+        libavformat-dev \
+        libavutil-dev \
+        libswresample-dev \
+        libswscale-dev \
+        libudev-dev \
+    && rm -rf /var/lib/apt/lists/*
```

We have packages that need `cmake`, so adding it here, allows us to remove it from the CI (only occurs on Python 3.9 at this time).

```suggestion
    apt-get update && apt-get install -y --no-install-recommends \
        libbluetooth-dev \
        tk-dev \
        uuid-dev \
        cmake \
        ffmpeg \
        libavcodec-dev \
        libavdevice-dev \
        libavfilter-dev \
        libavformat-dev \
        libavutil-dev \
        libswresample-dev \
        libswscale-dev \
        libudev-dev \
    && rm -rf /var/lib/apt/lists/*
```